Lib/multiprocessing/heap.py

```diff
@@ -227,7 +230,7 @@ def free(self, block):
 
     def malloc(self, size):
         # return a block of right size (possibly rounded up)
-        assert 0 <= size < sys.maxsize
+        assert 0 <= size < sys.maxsize, "Size {0:n} out of range".format(size)
```

Comment:
An OverflowError would be better here, IMO.

Reply:
OK. I did a ValueError if it was below 0, since that doesn't make sense to be an OverflowError.